pkg/operation/seed/seed.go

```diff
@@ -155,3 +178,9 @@ func (s *Seed) CheckMinimumK8SVersion() error {
 	}
 	return nil
 }
+
+// resourceQuantity returns a resource quantity based on number or nodes, resource per node,
+// node weight and base resource quantity.
+func resourceQuantity(baseResource, nodeCount, resourcePerNode, weight float64, SIUnit string) string {
```

Comment:
Could you move that function into another package (maybe `common`) so that we can reuse it also for other deployments until we can integrate the VPA?